doc/2-interface/song-info.md

```diff
@@ -45,7 +45,7 @@ _**Divider**:_ Changes the effective tick rate. a tick rate of 60Hz and a diviso
 **Highlight**: sets the pattern row highlights:
 - the first value represents the number of rows per beat.
 - the second value represents the number of rows per measure.
-- these don't have to line up with the music's actual beats and measures. set them as preferred for tracking. _Note:_ These values are used for the metronome and calculating BPM.
+- these don't have to line up with the music's actual beats and measures. set them as preferred for tracking. _Note:_ these values are used for the metronome and calculating BPM.
```

Comment:
The proper formatting for notes isn't correct.
Indent, then start a list with "note:" (no italic).
For example:
```
- 
  - note: